hera_cal/abscal.py

```diff
@@ -1020,8 +1020,8 @@ def omni_abscal_arg_parser():
     a.add_argument("--phase_slope_cal", default=False, action='store_true', help='perform frequency-indepdendent phase slope calibration')
     a.add_argument("--abs_amp_cal", default=False, action='store_true', help='perform absolute amplitude calibration')
     a.add_argument("--TT_phs_cal", default=False, action='store_true', help='perform Tip-Tilt phase slope calibration')
-    a.add_argument("--TT_phs_max_iter", type=int, default=100, help="maximum number of iterations of TT_phs_cal allowed")
-    a.add_argument("--TT_phs_conv_crit", type=float, default=1e-6, help="convergence criterion in Delta g / g for stopping iterative TT_phs_cal")
+    a.add_argument("--phs_max_iter", type=int, default=100, help="maximum number of iterations of phase_slope_cal or TT_phs_cal allowed")
+    a.add_argument("--phs_conv_crit", type=float, default=1e-6, help="convergence criterion in Delta g / g for stopping iterative phase_slope_cal or TT_phs_cal")
```

Comment:
is there a good reason why the `phs_conv_crit` is being used for both the global phase slope and the freq-dependent phase slope? If one is much slower than another, as you suggested, it might make sense to have two different knobs for these algorithms? However, I'm fine w/ keeping as is and adjusting if we see that in practice we find we need to..

Reply:
I'm just trying to avoid a huge proliferation of parameters and it seemed pretty odd to set these differently, to be honest. The one you want accurate is the final TT_phs, but the global slope is faster, so there's no real savings if you speed up the faster one.